docs/website/optimizations.md

```diff
@@ -304,3 +304,46 @@ methods with the `[BindingImpl (BindingImplOptions.Optimizable)]` attribute.
 It is always enabled by default (when the linker is enabled).
 
 The default behavior can be overridden by passing `--optimize=[+|-]dead-code-elimination` to mtouch/mmp.
+
+Optimize calls to BlockLiteral.SetupBlock
+-----------------------------------------
+
+The Xamarin.iOS/Mac runtime needs to know the block signature when creating an
+Objective-C block for a managed delegate. This might be a fairly expensive
+operation, and this optimization will calculate the block signature at build
+time, and modify the IL to call a SetupBlock variation that takes the
+signature as an argument instead, avoiding the need for calculating the
+signature at runtime.
+
+Benchmarks show that this speeds up calling a block 10-15x.
```

Comment:
Missing "by"?

"Benchmarks show that this speeds up calling a block by 10-15x."

Or maybe "Benchmarks show that this speeds up calling a block by a factor of 10 to 15."